package.json

```diff
@@ -19,9 +21,12 @@
   },
   "dependencies": {
     "chalk": "^2.3.2",
+    "license-checker": "^25.0.1",
     "lodash": "^4.17.5",
     "minimist": "^0.1.0",
-    "nlf": "^1.4.3"
+    "nlf": "^1.4.3",
+    "request": "^2.88.0",
+    "request-promise": "^4.2.2"
```

Comment:
Might consider `axios` in the future (https://github.com/axios/axios). `request` is old news - `axios` is promise based too